### PR TITLE
Set up Redux

### PR DESCRIPTION
**Pull request by lucabenaventew**
# Project Updates :green_circle: 
- [ ] Structure your application files using a "feature folder" approach
- [ ] Define action types for adding and removing a book.
- [ ] Set the initial state to be an empty array of books.
- [ ] Export Action Creators for your actions.
- [ ] Write your reducer and export it as default.
- [ ] Define an action type for checking the status.
- [ ] Set the initial state to be an empty array of categories.
- [ ] Export Action Creators for your actions.
- [ ] Write your reducer and export it as default
- [ ] Importing the necessary methods from Redux.
- [ ] Combine both reducers into a root reducer by using the combineReducers function.
- [ ] Use the root reducer to create your store.

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      ];
+    case REMOVED:
+      return state.splice(action.id, 1);
+    default: return state;
+  }
+};
+
+export function addBook() { return { type: ADDED }; }
+export function removeBook() { return { type: REMOVED }; }
+export default bookReducer;
```
Comment (cynthiainga) [inline src/redux/books/books.js:21]: - [ ] Good job so far, but I can't add or remove a book to your bookstore, please kindly fix that 🙃 🙃

File: src/redux/categories/categories.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const CHECKED = 'app/reducer/CHECKED';
+
+const checkReducer = (state = [], action) => {
+  switch (action.type) {
+    case CHECKED:
+      return 'Under construction';
+    default: return state;
+  }
+};
```
Comment (cynthiainga) [inline src/redux/categories/categories.js:9]: - [ ] Also when I click on the `checkstatus` button nothing happens, please kindly it as well 😉

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @lucabenaventew, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] All linters are passing ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Used correct gitflow ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comments under the review 👇 👇

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by lucabenaventew** [comment]
The Project requirement is not asking yet to use redux in react components, kindly accept my code because the project requirement is just set up redux.
The next project update does implement redux in react components

**Review by brytebee — APPROVED** [review]
Hi, @lucabenaventew👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great communication with the reviewer.✔️
- App functions as it should.✔️

##

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 

##

###  **Optional Suggestions:**
- N/A

------

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**